scripts/spack/packages/tribol/package.py

```diff
@@ -0,0 +1,300 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from spack.spec import UnsupportedCompilerError
+
+import os
+import socket
+from os.path import join as pjoin
+
+
+def get_spec_path(spec, package_name, path_replacements={}, use_bin=False):
+    """Extracts the prefix path for the given spack package
+       path_replacements is a dictionary with string replacements for the path.
+    """
+
+    if not use_bin:
+        path = spec[package_name].prefix
+    else:
+        path = spec[package_name].prefix.bin
+
+    path = os.path.realpath(path)
+
+    for key in path_replacements:
+        path = path.replace(key, path_replacements[key])
+
+    return path
+
+# NOTE: Cannot inherit from builtin until tribol becomes a builtin spack package within spack itself.
+
+class Tribol(CachedCMakePackage, CudaPackage):
+    """Tribol is an interface physics library."""
+
+    homepage = "https://lc.llnl.gov/gitlab/tribol/tribol"
+    git      = "ssh://git@czgitlab.llnl.gov:7999/tribol/tribol.git"
```

Comment:
These arent correct.

Reply:
should it be:
```
    homepage = "https://lc.llnl.gov/gitlab/tribol/tribol"
    git      = "https://lc.llnl.gov/gitlab/tribol/tribol.git"
```
?

or should we point to their github

Reply:
update: the links should be correct now. i moved the commit to one on their github. i also removed the patch since the changes are in the proj itself.

https://github.com/LLNL/Tribol/commit/c62690d8c21f94110d1f97d50c89fa87d142bb59